specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the specific branch in the repository."
+          },
+          {
+            "name":"configPath",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Configuration path of the file."
+          },
+          {
+            "name":"apiTag",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "API Tag to be validated."
+          },
+          {
+            "name":"commitHash",
+            "in": "body",
+            "type": "string",
+            "description": "Commit hash to be validated."
+          },
+          {
+            "name":"mode",
+            "in": "query",
+            "type": "string",
+            "description": "Mode"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/LinterValidationResult",
+              "description": "Linter Result"
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/LinterValidationResult",
+              "description": "Linter Result"
+            }
+          }
+        }
+      },
+      "x-ms-long-running-operation": true
+    },
+    "/tools/validation/semantic": {
+      "post": {
+        "tags": [
+          "semantic"
+        ],
+        "operationId": "Tools_Semantic",
+        "description": "Get Semantic validation result.",
+        "parameters": [
+          {
+            "name":"files",
+            "in": "body",
+            "type": "array",
+            "required": true,
+            "items": {
+              "type": "string"
+            },
+            "description": "List of files."
+          },
+          {
+            "name":"commitHash",
+            "in": "body",
+            "type": "string",
+            "description": "Commit hash to be validated."
+          },
+          {
+            "name":"validationId",
+            "in": "body",
+            "type": "string",
+            "description": "Validation Id."
+          },
+          {
+            "name":"githubToken",
+            "in": "body",
+            "type": "string",
+            "description": "Github Token"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
```

Comment:
please update descriptions in later commit, we don't want to have empty descriptions.